feat: add article on clearing app storage

JaspalSuri opened the pull request:
This is my first (significant) contribution to the docs repo and I'm not entirely familiar with the process, so feel free to let me know what needs to be corrected.

JaspalSuri committed:
feat: add article on clearing app storage

This is my first (significant) contirbution to the docs repo and I'm not entirely familiar with the process, so feel free to let me know what needs to be corrected.

theodorechu commented:
Hi Jaspal, thanks for putting this together. A few notes:
- I think it is probably better to make Troubleshooting a subsection of Usage, Listed, Self-hosting, or another main section to have its contents focused on the main topic instead of having a general Troubleshooting section.
- In line 19, we don't need to repeat the title.
- In line 41, we should follow the convention of **bolding** where to tap instead of putting the steps in a `code` snippet.
- It's probably better to title the doc according to the problem that we are trying to solve, such as "Troubleshooting issues signing in" instead of the solution "How to reset the apps?"
- The steps on this doc are quite technical, but it looks like the intended audience is users of the app. Following the steps seems like a huge hassle that users shouldn't have to go through. Perhaps this issue needs to be solved on the app level rather than something we normalize in the docs?

JaspalSuri committed:
remove troubleshooting section

JaspalSuri committed:
remove troubleshooting section from sidebar

JaspalSuri committed:
fix: remove redundant title on line 19

JaspalSuri committed:
fix: emphasize/bold steps in iOS' Settings app

JaspalSuri committed:
change: update article body

This contains the latest version and contains some improvements inherited from the commit.

JaspalSuri commented:
Hi @TheodoreChu,

Thanks for taking the time to review the commit. I apologize for the delay in getting back to you.

Given your last point, it's probably best to have this as an _unlisted_ article in the sense that it can be shared easily while not being prominently featured, so I've removed references to it from the `sidebar` and `config` files. 

I've applied the other changes that you've recommended in separate commits, except for renaming the title. I think that it should remain generic as the issue and use case can vary. 

Edit: should I keep or remove the sidebar label?

theodorechu committed:
fix: update formatting and language

- remove extra space after numbers
- move colon : into bold in `**please note:**`
- add period to the end of the sentences in numbered lists
- change **Warning** into an admonition
- updated meta description
- changed & to and

theodorechu committed:
fix: create second sidebar for Troubleshooting

theodorechu commented:
Thanks, Jaspal. I updated the formatting and some of the language and created a second sidebar for Troubleshooting. 👍🏼
